samcli/commands/validate/lib/sam_template_validator.py

```diff
@@ -73,52 +75,7 @@ def is_valid(self):
             ) from e
 
     def _replace_local_codeuri(self):
```

Comment:
My intention for leaving them as is is because I wanted to leave `sam_template_validator` as close to how it was originally formatted before I made my changes to it. My intention was not to replace those methods, but just to extract them so I could get the data from them. If you still believe that they should be removed, let me know and I will do that.

Reply:
you already moved the method to be used from a new module so I think you should remove the old one, and use the new method instead. If the old method is used anywhere else in the code, I think it should be refactored to use the new method.